### PR TITLE
Filter README.md out when listing files

### LINKED ISSUE #230: Fix the `README.md` problem at experiments during building
Although successfull, the [build process in Travis](https://travis-ci.com/qiskit-community/community.qiskit.org/builds/131399369) throws an error related to the `README.md` at `~/content/experiments/`, probably related to trying to parse it as an actual experiment.

### PR DESCRIPTION
**Pull request by delapuente**
Fix #230

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -174,12 +174,16 @@ const config: Configuration = {
...
-        .filter(filename => path.extname(filename) === '.md')
+        .filter(isMarkdownContent)
         .map(filename => `/events/${path.parse(filename).name}`)
       const experiments = fs.readdirSync(path.resolve(__dirname, 'content', 'experiments'))
-        .filter(filename => path.extname(filename) === '.md')
+        .filter(isMarkdownContent)
         .map(filename => `/experiments/${path.parse(filename).name}`)
       return events.concat(experiments)
+
+      function isMarkdownContent (filename: string): boolean {
```
Comment (y4izus) [inline nuxt.config.ts]: Could you change the function name to something that represent what is doing? It's called `isMarkdownContent` but it check also if it's the `README` file

File: nuxt.config.ts [hunk 2]
```diff
@@ -174,12 +174,16 @@ const config: Configuration = {
     dir: 'public',
     routes: (function () {
       const events = fs.readdirSync(path.resolve(__dirname, 'content', 'events'))
-        .filter(filename => path.extname(filename) === '.md')
+        .filter(isMarkdownContent)
```
Comment (y4izus) [inline nuxt.config.ts]: The way to get the events and the experiments is the same, only changing the string `events` to `experiments`. Could you create a function with the functionality in common?

Comment (delapuente) [reply nuxt.config.ts]: I considered it non related to this PR but I will do when adressing the other one.

File: nuxt.config.ts (hunk 1 above)
Comment (delapuente) [reply nuxt.config.ts]: The idea was to highlight the `README` is not content, thus the name. I'll give another take.

**Comment by delapuente** [comment]
@y4izus ready for another look!

**Pull request merged**